### PR TITLE
[Woo POS] Adapt the UI to be usable on smaller devices

### LINKED ISSUE #11937: [Woo POS] Adapt the UI to be usable on smaller devices

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #11937

### Description

Adapts the UI to be more usable on smaller screens

### Testing information

Test the changes on different screen size:
* Notice that it looks better on the smallest supported screens
* Make sure no changes are done on the big tablets

### Images/gif

image
image
image

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 300 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosSizes.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package com.woocommerce.android.ui.woopos.common.composeui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.woocommerce.android.ui.woopos.util.ext.getLongestScreenSideDp
+
+@Composable
+fun Dp.toAdaptivePadding(): Dp {
```
Comment (backwardstruck) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosSizes.kt:10]: Nice way to ensure padding is adjusted based on screen size.

**Review by backwardstruck — APPROVED** [review]
Code changes look good. 

Testing on small screen tablet:


Screenshot 2024-07-08 at 17 13 32


Large screen tablet testing:

Screenshot 2024-07-08 at 16 54 05

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosSizes.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosSizes.kt:10]: @samiuelson wdyt about this approach? That's kinda naive, but it seems to be working

Also, if we go with this, we should not forget to use that on all the paddings

**Pull request merged**